src/main/java/com/example/planit/engine/HolidaysEngine.java

```diff
@@ -101,7 +96,7 @@ public static List<Event> handleHolidaysInFullDaysEvents(List<Event> fullDayEven
      * @param year    the year we want the date of the holidays
      * @return string that return contain the full url to get request
      */
-    private static String createUrlForCurrentCountryAndYear(String holidaysApiKey, String country, int year) {
+    public static String createUrlForCurrentCountryAndYear(String holidaysApiKey, String country, int year) {
         return UrlRequest + "?" + "api_key=" + holidaysApiKey + "&country=" + country + "&year=" + year;
     }
```

Comment:
I think this method can be better written. maybe with `UrlBuilder`.  Do you want to do that? @oshriFadlon1  I think it's your method.

Reply:
you are right, this is indeed much better readable code, It's very strange how a former student of Guy Ronens Course manages to write such code. shame.